### ISSUE TITLE
[Use case]: load the core-business ontology

### ISSUE THREAD

**Issue by ticapix** [SEMICeu/Core-Business-Vocabulary#47]
### Use case name

load the core-business ontology

### Please insert the status of the use case

Tested

### Use case creator

Pierre Gronlier

### Stakeholders

List of stakeholders
- Riphixel (company)

### Problem statement

Probleme statement:

We can't load the ontology by resolving `http://www.w3.org/ns/legal#` nor pre-fetch a file directly from this repository.

(json context and shacl file are provided, but not a ontology file defining for example `owl:Class`, `owl:DatatypeProperty`, ...

Or at least, I couldn't find it

### Existing approaches

Existing approaches:

When we want to use W3C vcard, we can load the ontology in a turtle format, from the base uri `http://www.w3.org/2006/vcard/ns#`

### Links

Link list

### Requirements

Requirements:

as a developer working with ontology framework, I would like to be able to load the ontology with a machine readable file.

### Related use cases

Related use cases

### Comments

Comments

**Comment by PerNordanlind** [comment]
I agree on this, it is an issue we face in a Swedish context and also in the Nordic collaboration project NSG&B. Would be super if this is addressed in next update!

**Comment by EmidioStani** [comment]
This issue has similarity with #49

All Core Vocabularies concepts are under http://data.europa.eu/m8g on which SEMIC has authority but a Core Vocabulary include concepts from other namespaces.
The [RDF representation ](https://semiceu.github.io/Core-Business-Vocabulary/releases/2.2.0/voc/core-business-ap.ttl)of the Core Business, that can be found from the HTML specification, include only concepts from http://data.europa.eu/m8g.

Do you need just Core Business ? Do you need all the concepts or only those from http://data.europa.eu/m8g ?

At the moment, it is possible to retrieve all concepts under m8g from the URL: http://data.europa.eu/m8g
